wallet/src/de/schildbach/wallet/ui/more/MoreFragment.kt

```diff
@@ -294,6 +296,18 @@ class MoreFragment : Fragment(R.layout.fragment_more) {
         }
     }
 
+    private fun retry(errorMessage: String) {
```

Comment:
The more screen will support retry that can allow the user to choose a new name under some circumstances.